pkg/plugins/policies/meshfaultinjection/api/v1alpha1/validator_test.go

```diff
@@ -207,7 +207,7 @@ from:
 			[]validators.Violation{
 				{
 					Field:   "spec.from",
-					Message: "must not be defined",
+					Message: "must not be defined when the scope includes a Gateway, exclude non-gateway resources or select only gateways and use spec.to",
```

Comment:
"exclude non-gateway" has the same meaning as "select only gateways" :) probably should be
```
must not be defined when the scope includes a Gateway, select only proxyType Sidecar or select only gateways and use spec.to
```